pom.xml

```diff
@@ -401,8 +403,174 @@
+                </property>
+            </activation>
+            <properties>
+                <isolate.platform.suffix></isolate.platform.suffix>
+            </properties>
+        </profile>
+        <!--
+        Profile for using isolated version of a guest language.
+        The profile is activated by `mvn -Pisolated`.
+        -->
+        <profile>
+            <id>isolated</id>
+            <properties>
+                <isolated.language.id>js</isolated.language.id>
+            </properties>
+            <dependencies>
+                <dependency>
+                    <groupId>org.graalvm.polyglot</groupId>
+                    <artifactId>js-isolate${isolate.platform.suffix}</artifactId>
+                    <version>${graalvm.version}</version>
+                    <type>pom</type>
+                </dependency>
+                <dependency>
+                    <groupId>org.graalvm.polyglot</groupId>
+                    <artifactId>js</artifactId>
+                    <version>${graalvm.version}</version>
+                    <type>pom</type>
+                    <scope>provided</scope>
+                </dependency>
+            </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <version>3.1.2</version>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <polyglot.engine.SpawnIsolate>${isolated.language.id}</polyglot.engine.SpawnIsolate>
+                            </systemPropertyVariables>
+                        </configuration>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>exec-maven-plugin</artifactId>
+                        <version>3.1.0</version>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                            </execution>
+                            <execution>
+                                <id>no-runtime-compilation</id>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <configuration>
+                                    <executable>${java.home}/bin/java</executable>
+                                    <arguments>
+                                        <argument>-Dpolyglot.engine.SpawnIsolate=${isolated.language.id}</argument>
+                                        <!-- We recommend running from the module-path by default.
+                                        But you can also switch to the class-path here.-->
```

Comment:
But you can also switch to the class-path here. > You can also switch to the classpath here.